apps/frontend/src/components/ResultCard.tsx

```diff
@@ -0,0 +1,63 @@
+import { useState } from 'react';
+
+export type CardProps = {
+  url: string;
+  price: number;
+  currency: Currency;
+  name_of_website: string;
+  image_url: string;
+  title: string;
+};
+type Currency = 'usd' | 'uah' | 'eur' | 'gbp' | 'cny' | 'jpy';
+
+const conversion: Record<Currency, string> = {
+  usd: '$',
+  uah: '₴',
+  eur: '€',
+  gbp: '£',
+  cny: '¥',
+  jpy: '¥',
+};
+
+export default function ResultCard({ url, price, currency, name_of_website, image_url, title }: CardProps) {
+  const goToWebsite = () => {
+    // window.location.href =  url;
+    window.open(url, '_blank');
+  };
+  const [isFullNameShown, doShowFullName] = useState(false);
```

Comment:
Ahh I think my comment on this one was a bit confusing. I wasreferring to naming only the property itself (the 1st item in the array). The 2nd is a setter function and the the convention is to name it `set{PropertyName}`.

```suggestion
  const [isFullNameShown, setIsFullNameShown] = useState(false);
```